### PR TITLE
Changes to support ppc64le

### PR DESCRIPTION
**Pull request by seth-priya**
Fixes for ppc64le, reference issue #43

### CODE REVIEW

File: lib/manageiq/rpm_build/generate_gemset.rb [hunk 1]
```diff
@@ -63,6 +63,8 @@ def populate_gem_home
 
           shell_cmd("gem env")
           shell_cmd("gem install mime-types -v 2.6.1")
+          shell_cmd("if [ $(uname -m) = ppc64le ]; then gem install sassc  -- --disable-march-tune-native; fi")
+          shell_cmd("if [ $(uname -m) = ppc64le ]; then gem install unf_ext -v '0.0.7.2' -- --with-cxxflags='-fsigned-char'; fi")
```
Comment (simaishi) [inline lib/manageiq/rpm_build/generate_gemset.rb:67]: Putting `if` outside might be better?

```
if RUBY_PLATFORM =~ /ppc64le/
  shell_cmd
  shell_cmd
end
```

File: rpm_spec/subpackages/manageiq-gemset [hunk 2]
```diff
@@ -13,9 +13,11 @@ Requires: openscap-scanner
 Requires: openssl
 Requires: nodejs
 Requires: sqlite
+
+%ifnarch ppc64le
 Requires: wmi
+%endif
 
-# For Miq IPMI (gems-pending)
```
Comment (simaishi) [inline rpm_spec/subpackages/manageiq-gemset:18]: Accidentally deleted?

Comment (seth-priya) [reply rpm_spec/subpackages/manageiq-gemset:18]: yes, updating :(

File: lib/manageiq/rpm_build/generate_gemset.rb (hunk 1 above)
Comment (seth-priya) [reply lib/manageiq/rpm_build/generate_gemset.rb]: done

File: rpm_spec/subpackages/manageiq-gemset (hunk 2 above)
Comment (seth-priya) [reply rpm_spec/subpackages/manageiq-gemset:18]: done

**Comment by seth-priya** [comment]
@simaishi implemented changes ..

**Comment by simaishi** [comment]
Looks good!

**Comment by sethpeterson** [comment]
@miq-bot add-label jansa/yes?

**Comment by simaishi** [comment]
Jansa backport details:

```
$ git log -1
commit d443e49cbbcf502134e670bd3622275525babe6e
Author: Satoe Imaishi <simaishi@redhat.com>
Date:   Mon Jul 20 14:02:48 2020 -0400

    Merge pull request #76 from seth-priya/master

    Changes to support ppc64le

    (cherry picked from commit f70f75face2c76e25f5ab177597862d9087bc616)
```

**Pull request merged**